data_tools/restore.py

```diff
@@ -1,14 +1,24 @@
+import argparse
+
 import firebase_client_wrapper as fcw
 
 import json
 import sys
 
-if (len(sys.argv) != 3):
-    print ("Usage python restore.py crypto_token backup_file")
-    exit(1)
 
-CRYPTO_TOKEN_PATH = sys.argv[1]
-BACKUP_PATH = sys.argv[2]
+parser = argparse.ArgumentParser(description="Restores datasets from a backup file generated by backup.py")
+
+parser.add_argument("--max-segment-size", type=int, required=False, default=sys.maxsize,
```

Comment:
I don't think we should allow this. I think the backup and restore tools need to backup and restore the segment sizes. These tools really shouldn't be doing anything than the dumbest possible reads and writes, anything else creates complicated risks

Reply:
Agreed.

This comment reads in a way that suggests you didn't see the PR description. In that case, could you check that nothing else suggested there is about to be dropped by merging?

Reply:
resolved in #234